src/plugins/data/public/public.api.md

```diff
@@ -641,6 +642,23 @@ export function getQueryLog(uiSettings: IUiSettingsClient, storage: IStorageWrap
 // @public (undocumented)
 export function getSearchErrorType({ message }: Pick<SearchError, 'message'>): "UNSUPPORTED_QUERY" | undefined;
 
+// Warning: (ae-missing-release-tag) "getSearchParamsFromRequest" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
Maybe mark this as @internal?

Reply:
@lizozom not sure, becase we use it at least in 2 different plugins: `data, vis_type_vega`